cmake/developer_package/api_validator/api_validator.cmake

```diff
@@ -149,5 +149,5 @@ endfunction()
 # ie_add_api_validator_post_build_step(TARGET <name>)
 #
 macro(ie_add_api_validator_post_build_step)
-    _ie_add_api_validator_post_build_step(${ARGV})
+    _ov_add_api_validator_post_build_step(${ARGV})
 endmacro()
```

Comment:
please, rename in this place as well https://github.com/openvinotoolkit/openvino/blob/fca742592aeffda01e54b0b0c95286087df3ad85/cmake/developer_package/api_validator/api_validator.cmake#L55

Reply:
Fixed